### DISCUSSION TITLE
Tolerance for testing column value equality

### DISCUSSION THREAD

**Discussion by eveyp** [rstudio/pointblank#325] [Ideas]
Would it be possible to specify a tolerance when testing if column values are equal to a specified value? This would be helpful to avoid floating point round off issues. 

It could be similar to how `testthat` has `expect_equal()` which allows for a tolerance and `expect_identical()` which requires the values to be exactly the same (https://testthat.r-lib.org/reference/equality-expectations.html).

**Comment by rich-iannone** [comment]
Yes! This is a great idea. Both for the `col_vals_equal()` and also the `col_vals_not_equal()` functions. Mind if I convert this into an issue (I see there is a new option for that). ----->>

**Comment by eveyp** [comment]
Thanks! Go for it!!

On Wed, Jun 30, 2021 at 12:24 PM Richard Iannone ***@***.***>
wrote:

> Yes! This is a great idea. Both for the col_vals_equal() and also the
> col_vals_not_equal() functions. Mind if I convert this into an issue (I
> see there is a new option for that). ----->>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rich-iannone/pointblank/discussions/325#discussioncomment-945055>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADR5GPGIU7L5Q4CBD5BJLZDTVNVOBANCNFSM47S5D7UA>
> .
>


-- 
eve perry (she/they)

**Reply by rich-iannone** [reply]
Thank you! Now in https://github.com/rich-iannone/pointblank/issues/326
